options/options_parser.cc

```diff
@@ -656,8 +660,17 @@ Status RocksDBOptionsParser::VerifyRocksDBOptionsFromFile(
     const std::vector<ColumnFamilyOptions>& cf_opts,
     const std::string& file_name, FileSystem* fs,
     OptionsSanityCheckLevel sanity_check_level, bool ignore_unknown_options) {
+  // We infer option file readhead size from log readahead size.
```

Comment:
I wonder whether we can use `paranoid_checks` to determine whether we verify options file after persisting it.

Reply:
Yes, we can do that. It would be a separate task though.